src/main/java/com/puppycrawl/tools/checkstyle/filters/SuppressionPatchFilter.java

```diff
@@ -69,7 +69,7 @@
     /**
      * Control if only consider added lines in file.
      */
-    private String strategy;
+    private String strategy = "newline";
```

Comment:
use constant
https://github.com/checkstyle/patch-filters/blob/6147cc63725fa64eef9fb1777e36ad4831fe2ee3/src/main/java/com/puppycrawl/tools/checkstyle/filters/LoadPatchFileUtils.java#L38

Reply:
done, is necessary to use enum class?

Reply:
I meant to reuse the constant from LoadPatchFileUtils. No need to create the same constant in each class.

But yes, Enum for all these strategies will be even better.

Reply:
done